### ISSUE TITLE
Provide a (multi-)selection as an alternative to the filter-input

### ISSUE THREAD

**Issue by sboekhoff** [eddyson-de/react-grid#160]
Especially for columns with a limited set of values (e.g. boolean, states, ...) it would be helpful to select one (or several) values from a dropdown menue instead of typing values to the filter-input. (For boolean values even this is not possible at the moment).

In my imagination the filter input (for certain columns) could be replaced by a multiselect (e.g. https://github.com/ehynds/jquery-ui-multiselect-widget):

![jquery-ui-multiselect-widget](https://cloud.githubusercontent.com/assets/10683690/20216131/24f7a90e-a819-11e6-9b8c-d9e1738a91ec.gif)

**Comment by jochenberger** [comment]
I don't think we should add a dependency on jQuery. If we can craft something using react-bootstrap (which we depend on already), I'm all for it.
We'll probably have to make the filter component exchangeable on a per-row basis. We could technically render different fields depending on the column key, which is passed into the Filter component, but I guess that will make things pretty complicated.

> For boolean values even this is not possible at the moment

Could you please create a separate issue for that?
